src/NuGet.Services.SearchService/NuGet.Services.SearchService.csproj

```diff
@@ -115,40 +115,40 @@
   </PropertyGroup>
   <Import Project="$(SignPath)\sign.targets" Condition="Exists('$(SignPath)\sign.targets')" />
   <ItemGroup>
-    <ThirdPartyBinaries Include="AnglicanGeek.MarkdownMailer.dll" />
-    <ThirdPartyBinaries Include="Autofac.dll" />
-    <ThirdPartyBinaries Include="Autofac.Extensions.DependencyInjection.dll" />
-    <ThirdPartyBinaries Include="Autofac.Integration.WebApi.dll" />
-    <ThirdPartyBinaries Include="Dapper.StrongName.dll" />
-    <ThirdPartyBinaries Include="dotNetRDF.dll" />
-    <ThirdPartyBinaries Include="Elmah.dll" />
-    <ThirdPartyBinaries Include="HtmlAgilityPack.dll" />
-    <ThirdPartyBinaries Include="ICSharpCode.SharpZipLib.dll" />
-    <ThirdPartyBinaries Include="json-ld.net.StrongName.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.Analyzers.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.Core.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.FastVectorHighlighter.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.Highlighter.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.Memory.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.Queries.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.Regex.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.SimpleFacetedSearch.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.Snowball.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Contrib.SpellChecker.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.dll" />
-    <ThirdPartyBinaries Include="Lucene.Net.Store.Azure.dll" />
-    <ThirdPartyBinaries Include="Markdig.dll" />
-    <ThirdPartyBinaries Include="MarkdownSharp.dll" />
-    <ThirdPartyBinaries Include="Newtonsoft.Json.dll" />
-    <ThirdPartyBinaries Include="Serilog.dll" />
-    <ThirdPartyBinaries Include="Serilog.Enrichers.Environment.dll" />
-    <ThirdPartyBinaries Include="Serilog.Enrichers.Process.dll" />
-    <ThirdPartyBinaries Include="Serilog.Extensions.Logging.dll" />
-    <ThirdPartyBinaries Include="Serilog.Sinks.ApplicationInsights.dll" />
-    <ThirdPartyBinaries Include="Serilog.Sinks.ColoredConsole.dll" />
-    <ThirdPartyBinaries Include="Serilog.Sinks.File.dll" />
-    <ThirdPartyBinaries Include="SerilogTraceListener.dll" />
-    <ThirdPartyBinaries Include="VDS.Common.dll" />
+    <ThirdPartyBinaries Include="AnglicanGeek.MarkdownMailer.dll" Visible="false" />
```

Comment:
This mirrors the CDN Redirect service: https://github.com/NuGet/NuGet.Jobs/blob/master/src/NuGetCDNRedirect/NuGetCDNRedirect.csproj#L145